docs/content/preview/develop/change-data-capture/using-logical-replication/key-concepts.md

```diff
@@ -26,6 +26,15 @@ In logical replication, the fundamental unit of data transmission is a transacti
 
 For more information, refer to [Replication slots](https://www.postgresql.org/docs/15/logicaldecoding-explanation.html#LOGICALDECODING-REPLICATION-SLOTS) in the PostgreSQL documentation.
 
+#### LSN type
+
+A Log Sequence Number (LSN) in YugabyteDB differs from what you may be accustomed to in PostgreSQL. In PostgreSQL, an LSN represents a specific 'location' in the WAL, and has significance that spans databases and replication slots. In YugabyteDB, an LSN uniquely identifies a change event, and the LSN is valid only within the context of a specific replication slot. Due to these differences, there are inherent limitations in how LSNs can be used.
```

Comment:
A Log Sequence Number (LSN) in YugabyteDB differs in its semantics from PostgreSQL.